perun-core/src/main/java/cz/metacentrum/perun/core/api/MembersManager.java

```diff
@@ -49,9 +50,11 @@
 	 * @throws VoNotExistsException
 	 * @throws PrivilegeException
 	 * @throws MemberAlreadyRemovedException
-	 * @throws GroupOperationsException
+	 * @throws WrongAttributeValueException
+	 * @throws AlreadyMemberException
```

Comment:
Again, deleting member (all members) shouldn't throw stuff like ``AlreadyMemberException``, try/catch it in Bl and throw ``ConsistencyErrorException``.